apptools/preferences/preferences_helper.py

```diff
@@ -134,7 +140,8 @@ def _get_value(self, trait_name, value):
 
         # If the trait type is 'Unicode' then we convert the raw value.
```

Comment:
Man, that's very unreliable.

Can we just make `ConfigObj(file_or_filename, encoding='utf-8')` instead? What breaks when we do that?